src/main/java/Deadline.java

```diff
@@ -0,0 +1,29 @@
+import java.lang.reflect.Array;
+
+public class Deadline extends Task {
+    protected Date byDate;
+    protected Time byTime;
+    protected String exactBy;
+
+    public Deadline(String description, String by) {
+        super(description);
+        this.exactBy = by;
+        String[] temp = by.split(" ");
```

Comment:
Can consider adding an exception handling mechanism here just in case string is not the correct format.